web/thesauruses/_meta/io.json

```diff
@@ -0,0 +1,243 @@
+{
+    "meta": {
+        "language": "language_id",
+        "language_version": "version.number",
+        "language_name": "Human-Friendly Language Name"
+    },
+    "categories": {
+        "Console IO": [
+            "write_line",
+            "write_line_with_new_line",
+            "read_line",
+            "read_char",
+            "clear_console_cutput",
+            "change_console_color"
+        ],
+        "File IO": [
+            "list_directory",
+            "read_directory_info",
+            "create_directory",
+            "delete_directory",
+            "rename_directory",
+            "move_directory",
+            "update_directory_permissions",
+            "read_line_of_file",
+            "read_lines_of_file",
+            "read_file_stream",
+            "create_file",
+            "delete_file",
+            "update_file",
+            "move_file"
+        ],
+        "Network IO": [
+            "send_a_get_request",
+            "send_a_post_request",
+            "add_headers",
+            "add_query_params",
+            "inbuilt_url_class",
+            "open_a_socket",
+            "close_a_socket",
+            "add_authorization_headers",
+            "add_cookies",
+            "read_cookies",
+            "read_headers"
+        ],
+        "Database IO": [
+            "open_connection",
+            "close_connection",
+            "execute_query_single_result_set",
+            "execute_query_multiple_result_sets",
+            "execute_non_query"
+        ],
+        "JSON": [
+            "serialize_json",
+            "deserialize_json",
+            "serialize_json_with_options",
+            "deserialize_json_with_options"
+        ],
+        "XML": [
+            "serialize_xml",
+            "deserialize_xml",
+            "serialize_xml_with_options",
+            "deserialize_xml_with_options"
+        ]
+    },
+    "io": {
+        "write_line": {
+            "name": "Write a line to the console",
+            "code": ""
+        },
+        "write_line_with_new_line": {
+            "name": "Write a line to the console with a return charactor",
+            "code": ""
+        },
+        "read_line": {
+            "name": "Read a string of user input from the console",
+            "code": ""
+        },
```

Comment:
You call this "read_line" but say "read a string...", should it maybe be "read a line of..."?